addon/templates/components/frost-navigation.hbs

```diff
@@ -17,4 +16,7 @@
   <!-- User Menu -->
   {{component menu}}
 </div>
-{{liquid-modal}}
+{{to-elsewhere
+  named='modal'
```

Comment:
I know this is technically an internal name but should it be more specific so as to not conflict with any other possible elsewhere 'modals'?